profiles/package.mask

```diff
@@ -175,16 +175,6 @@ dev-python/pydiff
 # as replacement. Bug #769017.
 app-i18n/man-pages-it
 
-# David Seifert <soap@gentoo.org> (2021-08-04)
-# Last release over 4 years ago, upstream pretty much dead, the
-# ecosystem has switched to dev-util/pkgconf, which is alive. Testing
-# and prefix bugs, blocks WANT_AUTOMAKE=1.12 removal.
-# (If you hit blockers, please run:
-# $ emerge --deselect dev-util/pkgconfig
-# This is necessary as your world file should not contain dependencies.)
-# Bug #245228, #632124, #691268, #767853, removal in 30 days.
-dev-util/pkgconfig
-
```

Comment:
I would leave the blocker, as it helps people with `dev-util/pkgconfig` in their world file

Reply:
makes sense, done ;)